src/main/webapp/webpack.config.js

```diff
@@ -14,7 +14,7 @@ try {
     console.error('\nFailed to load Geppetto Configuration')
 }
 
-var publicPath = ((geppettoConfig.contextPath == '/') ? geppettoConfig.contextPath : "/" + geppettoConfig.contextPath + "/") + "geppetto/build/";
+var publicPath = ((geppettoConfig.contextPath == '/') ? geppettoConfig.contextPath : "/" + geppettoConfig.contextPath) + "geppetto/build/";
```

Comment:
Same here... What if contextPath does not contain a slash at the very end? Maybe we can have a function a bit smarter that checks if actually a slash is needed and makes a proper join? @rodriguez-facundo what do you think?

Reply:
done